app/src/main/res/values/strings.xml

```diff
@@ -102,6 +103,12 @@
     <string name="connect_status_provider_success">\%s was successfully connected!</string>
     <string name="connect_status_provider_success_description">Your Service Provider has been successfully connected to the Authenticator App. You will receive an authorization request every time you make an action.</string>
 
+    <string name="action_feature_title">Successful authentication</string>
```

Comment:
it is not `authentication` feature. fix resource

Reply:
I agree, that's why I wrote `action_feature_title`

Reply:
User will see resource value not your key

Reply:
https://github.com/saltedge/sca-authenticator-android/issues/81#issuecomment-583360992